### DISCUSSION TITLE
Windows Server (DCs) & Ubuntu/Debian Security Updates

### DISCUSSION THREAD

**Discussion by lockstock27** [alan-turing-institute/data-safe-haven#1675] [Q&A] [answered]
Hi,
Once deployed do the DCs & Ubuntu servers receive security updates?

Thanks

**Comment by JimMadge** [comment, marked as answer]
Hi @lockstock27,

Yes, [Azure automation](https://learn.microsoft.com/en-us/azure/automation/overview) is used to install updates to VMs. Fore the Ubuntu machines, this is through another machine running [Squid](http://www.squid-cache.org/).

The update management view in the automation account shows if updates have failed to apply, so this is worth checking or setting up alerts for.
